buf.gen.yaml

```diff
@@ -19,11 +19,11 @@ plugins:
     # https://grpc-ecosystem.github.io/grpc-gateway/docs/mapping/customizing_openapi_output/#merging-output
   - plugin: grpc-gateway-ts
     out: gen/api-ts
-  - name: gateway-client
-    path: protoc-gen-grpc-gateway-client
-    out: gen/protos
-    opt:
-      - paths=source_relative
+  # - name: gateway-client
```

Comment:
@seanmcgary does this package exist - i was not able to find it - commenting it helped generate protos